### PR TITLE
Merge v1.5

### PR DESCRIPTION
**Pull request by akombeiz**
Anpassungen an AKTIN DWH V1.5 (nur debian)
Integrationtests laufen alle durch
EOF newline wurde in jeder Funktion manuell geprüft
Kommentare und Funktionsnamen wurden angepasst

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,4 +2,3 @@
 set -euo pipefail
 . debian/build.sh "$1" "$2"
 . docker/build.sh "$1" "$2" "full"
-
```
Comment (simonkurka) [inline build.sh:5]: newline on EOF like in i2b2-pkg

File: common/build.sh [hunk 2]
```diff
@@ -22,53 +22,63 @@ function dwh_j2ee() {
...
-	datasource_postinstall "/usr/share/${PACKAGE}/datasource"
+	download_dwh_j2ee "/opt/wildfly/standalone/deployments"
+	config_apache2_proxy "/etc/apache2/conf-available" "localhost"
+	create_aktin_dir "/var/lib/aktin"
+	create_aktin_dir "/var/lib/aktin/import"
+	move_aktin_properties "/etc/aktin"
+	move_aktin_importscripts "/var/lib/aktin/import-scripts"
+	move_database_for_postinstall "/usr/share/${PACKAGE}/database"
+	move_database_update_for_postinstall "/usr/share/${PACKAGE}/database-update"
+	move_datasource_for_postinstall "/usr/share/${PACKAGE}/datasource"
```
Comment (simonkurka) [inline common/build.sh:83]:
```suggestion
	copy_aktin_properties "/etc/aktin"
	copy_aktin_importscripts "/var/lib/aktin/import-scripts"
	copy_database_for_postinstall "/usr/share/${PACKAGE}/database"
	copy_database_update_for_postinstall "/usr/share/${PACKAGE}/database-update"
	copy_datasource_for_postinstall "/usr/share/${PACKAGE}/datasource"
```

File: debian/postinst [hunk 3]
```diff
@@ -62,11 +72,16 @@ else
...
+done
+
+# deploy aktin datasource
 sed -e "s/__HOST__/$host/g" \
     -e "s/__PORT__/$port/g" \
     /usr/share/$DPKG_MAINTSCRIPT_PACKAGE/datasource/aktin-ds.xml > /opt/wildfly/standalone/deployments/aktin-ds.xml
 
-chown -R wildfly:wildfly /var/lib/aktin
-
+chown -R wildfly:wildfly /var/lib/aktin /opt/wildfly/standalone/deployments
```
Comment (simonkurka) [inline debian/postinst:86]: `chown -R`on deployments folder is too much. There're only two specific files which are placed in the folder by this package. Everything else is out of scope of this package.
```suggestion
chown -R wildfly:wildfly /var/lib/aktin /opt/wildfly/standalone/deployments/aktin-ds.xml /opt/wildfly/standalone/deployments/dwh-j2ee-__DWH_J2EE_VERSION__.ear
```
Must replace `__DWH_J2EE_VERSION__` during build preparation.

File: resources/aktin.properties [hunk 4]
```diff
@@ -51,6 +51,9 @@ import.data.path=/var/lib/aktin/import
 import.script.path=/var/lib/aktin/import-scripts
 import.script.timeout=10800000
 
+# data path for dwh-update service
+update.data.path=/var/lib/aktin/update
```
Comment (simonkurka) [inline resources/aktin.properties:55]: is there any reason, why the user might change this?

File: resources/versions [hunk 5]
```diff
@@ -1,2 +1 @@
 VDWH_J2EE=1.4
```
Comment (simonkurka) [inline resources/versions:1]:
```suggestion
VDWH_J2EE=1.5
```
For 1.5 update, this should be changed?!

**Review by simonkurka — CHANGES_REQUESTED** [review]
without tests

File: resources/aktin.properties (hunk 4 above)
Comment (akombeiz) [reply resources/aktin.properties:55]: In terms of style, each folder of /var/lib/aktin is configurable via aktin.properties. Basically, the user should not mess around with the update service, so the path could also be included in the source code

File: resources/versions (hunk 5 above)
Comment (akombeiz) [reply resources/versions]: Set to 1.5rc1

**Review by simonkurka — APPROVED** [review]

**Pull request merged**